command/run.go

```diff
@@ -32,10 +32,19 @@ Examples:
 		 > cat commands.txt | s5cmd {{.HelpName}}
 `
 
+var runCommandFlags = []cli.Flag{
+	&cli.BoolFlag{
+		Name:    "dry-run-all",
```

Comment:
why not `--dry-run`?

Reply:
normally, parent context flag values can be accessed from the child context. but if same flags are used they cannot, child context flag overrides to empty even if it is not set.

so in this case if  `--dry-run` flag is used for both `run` command and `mv`/`cp`. setting this flag in `run` has no effect for `mv`/`cp`

Reply:
hmm ok.